src/Codeception/Module/Db.php

```diff
@@ -517,6 +517,19 @@ protected function proceedSeeInDatabase($table, $column, $criteria)
         return $sth->fetchColumn();
     }
 
+    public function grabColumnFromDatabase($table, $column, array $criteria = [])
```

Comment:
Please add documentation in docblock for this method.
The similar method `grabFromDatabase` is documented in its interface https://github.com/Codeception/Codeception/blob/2.3/src/Codeception/Lib/Interfaces/Db.php
Use its documentation as an example and write some docs here. Usage examples with code would be nice.